website/docs/index.html.markdown

```diff
@@ -21,18 +21,22 @@ resources.
 In order to use the provider you need to obtain the authentication
 token from the Cloud Service Provider by providing the org scoped API token.
 The provider client uses Cloud Service Provider (CSP) API
-to exchange this org scoped API token for user access token.
+to exchange this org scoped API token/OAuth App Id and Secret for user access token.
 
-Note that in all of the examples you will need to update the `api_token` and `org_id` settings
-in the variables.tf file to match those configured in your VMC environment.
+Note that in all the examples you will need to update the `api_token` (or `client_id` and `client_secret`)
```

Comment:
I do not think you need the brackets. I guess a lot of readers will be developers and can benefit the brackets for logical grouping. It is not easy to read them though.

Reply:
Addressed.